src/context/volume-context.js

```diff
@@ -1,17 +1,21 @@
 import { createContext, useContext, useReducer } from "react";
+import config from "../assets/config.json";
 
 const VolumeContext = createContext();
-const pathPrefix = "./assets/models/";
 
 // Custom component to provide the Volume context
 function VolumeProvider(props) {
   const [state, dispatch] = useReducer(volumeReducer, {
-    options: {
-      measurement: "salt",
-      season: "summer",
-      tide: "low",
+    selection: {
+      season: config.season[0],
+      tide: config.tide[0],
+      measurement: config.measurement[0],
     },
-    path: `${pathPrefix}summer_low_salt.png`,
+    slices: 55,
+    extension: ".png",
+    x_spacing: 2.0,
```

Comment:
these should read from the config as well

Reply:
I think that's the default value.  @RobertGemmaJr ??